features/number.yml

```
@@ -0,0 +1,30 @@
+name: Number
+description: The Number object is used to store number and manipulate floating-point numbers, like 42, or -4.201
```

```suggestion
description: The `Number` object represents floating-point numbers, such as 42 or -4.201. JavaScript numbers can also represent `Infinity` (boundless negative and positive values) or `NaN` (not a number, as in `0 * Infinity`).
```

---

Actually, on second thought, because JavaScript's primitive object wrappers are weird:

```suggestion
description: The number type (and `Number` object) represents floating-point numbers, such as 42 or -4.201. JavaScript numbers can also represent boundless negative and positive values as `Infinity` or not-a-number as `NaN` (as in `0 * Infinity`).
```